docs/secure/Idp_integration.md

```diff
@@ -0,0 +1,154 @@
+
+Percona Everest uses [OpenID Connect](https://auth0.com/docs/authenticate/protocols/openid-connect-protocol){:target="_blank"} (OIDC) Protocol to integrate with external Identity Providers (IdP).
+
+
+!!! note "Note"
+    To integrate IdP with Percona Everest, first, [install Percona Everest](../install/installEverest.md) and then [configure OIDC](#configure-oidc-on-the-providerss-side) on the IdP's side as well as the [Percona Everest side](#configure-oidc-on-percona-everest-side).
+
+## Configure OIDC on the providers's side
+
+- **Proof Key for Code Exchange (PKCE)**: When setting up the Provider side, it's important to configure an application specifically for [PKCE](https://auth0.com/docs/get-started/authentication-and-authorization-flow#authorization-code-flow-with-proof-key-for-code-exchange-pkce-){:target="_blank"} authorization.
+
+- **Redirect URIs**: 
+
+    * Sign-in redirect URIs should point to `<EVEREST_URL>/login-callback`.
+    * Sign-out redirect URIs should point to `<EVEREST_URL>`.
+ 
+        !!! info "Important" 
+            IdP providers often require a secure connection (HTTPS). Therefore, the provider might require your `<EVEREST_URL>` to be based on HTTPS. If your provider requires this, consider adding a reverse proxy in front of Percona Everest to provide such functionality.
+
+### Example: OIDC configuration on the providers's side
+
+Below is an example of the configuration for the IdP provider Okta. 
+
+Other popular IdPs include Microsoft Azure Active Directory, Auth0, Google Identity Platform, etc.
+
+
+??? example "OKTA"
+
+    1. Sign in to your Okta organization as a user with administrative privileges.
+        
+    2. In the **Admin** Console, go to **Applications → Applications** and click **Create App Integration**.
+
+    3. On the **Create a new app integration page**, set the following:
+        
+        - Sign-in method - **OIDC - OpenID Connect**
+        - **Application type** - **Signle-Page Application**, and click **Next**.
+
+        ![!image](../images/OKTA_admin_console.png)
+
+    4. Set the following fields:
+
+        a. App integration name - any value
+
+        b. Sign-in redirect URIs - `<EVEREST_URL>/login-callback`
+
+        c. Sign-out redirect URIs - `<EVEREST_URL>`
+
+        d. Click **Save**.
+
+        e. Copy the `clientID` of the created app.
+
+        f. Navigate to **Security → API → Authorization Servers** and copy the `issuerURL` you’d like to use for the Everest authorization. 
+
+        !!! note "Note"
+            Okta allows the use of HTTP for development purposes and in cases where the Admin explicitly permits it.
+
+        ![!image](../images/sso_aap_integration.png)
+
+
+
+## Configure OIDC on the Percona Everest's side
+
+You can configure OIDC on the Percona side using the flag or the wizard.
```

Comment:
Maybe we should have a quick mention about the Issuer URL and Client ID. We should mention that these were the fields we asked the user to copy in the previous section.